packages/scss/src/_typo.scss

```diff
@@ -7,39 +7,39 @@ html {
 }
 
 body {
-	background: $greyBackground;
-	color: $grey;
+	background: #F5F5F5;
+	color: _get($luTheme, "palette.colors.greyStandard");
 	font-family: "Source Sans Pro", Tahoma, sans-serif;
-	font-size: $font-standard;
-	line-height: $font-line-standard;
+	font-size: _get($luTheme, "font.size.standard");
+	line-height: _get($luTheme, "font.line.standard");
 	vertical-align: middle;
 }
 
 h1 {
-	font-size: $font-biggest;
+	font-size: _get($luTheme, "font.size.biggest");
 	font-weight: 600;
-	line-height: $font-line-biggest;
+	line-height: _get($luTheme, "font.line.biggest");
 	margin: 0 0 1.7rem;
```

Comment:
I personally never trust anything or anyone. I always live in fear of having something changing body font-size, thus destroying my measures.